bin/run.js

```diff
@@ -129,12 +129,14 @@ function runKaba (opts, isVerbose)
         logger.log("kaba started");
         const start = process.hrtime();
         const cliConfig = new CliConfig(opts);
+        let workingDirectory = process.cwd();
```

Comment:
Kaba already has a `cwd`: https://github.com/Becklyn/kaba/blob/master/lib/Kaba.js#L39

Please use this exact value and pass it out through the build config, as mentioned in https://github.com/Becklyn/kaba/pull/50#discussion_r256862775

Reply:
Well, I was just re-using existing code from this file :)

I've added a getter in `Kaba` to access its `cwd`.